### PR TITLE
Add additional metrics to new observatorium tenant

### PR DESCRIPTION
**Pull request by RiRa12621**
This replicates https://github.com/openshift/managed-cluster-config/pull/695 to current used config

### CODE REVIEW

File: deploy/cluster-monitoring-config-uwm/cluster-monitoring-config.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ data:
     # so that killswitch instructions point to the right file!
       remoteWrite:
         - url: http://token-refresher.openshift-monitoring.svc.cluster.local
+          max_samples_per_send: 100
```
Comment (jharrington22) [inline deploy/cluster-monitoring-config-uwm/cluster-monitoring-config.yaml:15]: @RiRa12621 where is this requirement coming from?

Comment (RiRa12621) [reply deploy/cluster-monitoring-config-uwm/cluster-monitoring-config.yaml:15]: Was a recommendation from the observatorium team

Comment (jharrington22) [reply deploy/cluster-monitoring-config-uwm/cluster-monitoring-config.yaml:15]: The configuration document from the observatorium team recommended this change.

**Pull request merged**